README.md

```diff
@@ -48,7 +48,7 @@ For development work the following specifications have worked well in the past:
 * m5.large
 * 60 GB disk
 
-In `dev` mode Docker containers will be built from source and the VM will run them during testing. (see [here](https://github.com/DataBiosphere/cgp-deployment/blob/feature/update-readme/boardwalk/README.md#development-mode) for more details). If work is planned only in *Boardwalk*, but not in common, and particularly if real certificates are needed during development, it is recommended to set up *Common* in `prod` mode, and *Boardwalk* in `dev` mode.
+In `dev` mode Docker containers will be built from source and the VM will run them during testing. (see [here](https://github.com/DataBiosphere/cgp-deployment/blob/feature/update-readme/boardwalk/README.md#development-mode) for more details). If work is planned only in *Boardwalk*, but not in *Common*, and particularly if real certificates are needed during development, it is recommended to set up *Common* in `prod` mode, and *Boardwalk* in `dev` mode.
```

Comment:
1. `will be built from source`
    * Since you are making the distinction further below between common and boardwalk, which is good, then you should probably clarify the difference here as well; only in Boardwalk does it build from source; for common, the difference between prod and dev is whether it issues fake certificates or not.
2. `the VM will run them during testing`
    * There is no VM running things. Unless you're referring to Docker, but Docker technically isn't a VM.
    * I don't think `during testing` adds anything.
3. `it is recommended to set up *Common*...`.
    * The setup for dev mode and prod mode is the same I believe, i.e., you answer the same questions for both. I would say `it is recommended to run *Common*...`
4. Your last sentence seems to be phrased a little awkwardly, to me. I would simplify to something like `Even if you run Boardwalk in dev mode, it is recommended that you run common in prod mode, otherwise you can run into various issues with the fake certificates that are issued in dev mode.`; something like that.

Reply:
Thanks for the comments. I addressed all 4 points. I think it's better now.